src/commands/force/source/deploy.ts

```diff
@@ -225,37 +225,79 @@ export class Deploy extends DeployCommand {
+      : `Components deployed:  ${data.numberComponentsDeployed}`;
+    this.ux.log('');
+    this.ux.log(`Status:  ${data.status}`);
+    this.ux.log(`jobid:  ${data.id}`);
+    this.ux.log(`Component errors:  ${data.numberComponentErrors}`);
+    this.ux.log(successfulComponentsMessage);
+    this.ux.log(`Components total:  ${data.numberComponentsTotal}`);
+    this.ux.log(`Tests errors:  ${data.numberTestErrors}`);
+    this.ux.log(`Tests completed:  ${data.numberTestsCompleted}`);
+    this.ux.log(`Tests total:  ${data.numberTestsTotal}`);
+    this.ux.log(`Check only: ${data.checkOnly}`);
+    this.ux.log('');
+  }
+
+  private progress(deploy: MetadataApiDeploy, progressBar = true): void {
     const startProgressBar = once((componentTotal: number) => {
       this.progressBar.start(componentTotal);
     });
-
     deploy.onUpdate((data) => {
-      // the numCompTot. isn't computed right away, wait to start until we know how many we have
-      if (data.numberComponentsTotal) {
-        startProgressBar(data.numberComponentsTotal + data.numberTestsTotal);
-        this.progressBar.update(data.numberComponentsDeployed + data.numberTestsCompleted);
-      }
-
-      // the numTestsTot. isn't computed until validated as tests by the server, update the PB once we know
-      if (data.numberTestsTotal && data.numberComponentsTotal) {
-        this.progressBar.setTotal(data.numberComponentsTotal + data.numberTestsTotal);
+      if (!progressBar) {
+        // If not progress bar. Print the status in normal formatt
+        this.printStatus(data);
+      } else {
+        // the numCompTot. isn't computed right away, wait to start until we know how many we have
+        if (data.numberComponentsTotal) {
+          startProgressBar(data.numberComponentsTotal + data.numberTestsTotal);
+          this.progressBar.update(data.numberComponentsDeployed + data.numberTestsCompleted);
+        }
+        // the numTestsTot. isn't computed until validated as tests by the server, update the PB once we know
+        if (data.numberTestsTotal && data.numberComponentsTotal) {
+          this.progressBar.setTotal(data.numberComponentsTotal + data.numberTestsTotal);
+        }
       }
     });
-
     // any thing else should stop the progress bar
     deploy.onFinish((data) => {
-      // the final tick of `onUpdate` is actually fired with `onFinish`
-      this.progressBar.update(data.response.numberComponentsDeployed + data.response.numberTestsCompleted);
-      this.progressBar.stop();
+      if (progressBar) {
+        // the final tick of `onUpdate` is actually fired with `onFinish`
+        this.progressBar.update(data.response.numberComponentsDeployed + data.response.numberTestsCompleted);
+        this.progressBar.stop();
+      } else {
+        // Prints the final Status
+        this.printStatus(data.response);
+      }
     });
 
     deploy.onCancel(() => {
-      this.progressBar.stop();
+      if (progressBar) this.progressBar.stop();
```

Comment:
does toolbelt print anything for these events without the progress bar?

Reply:
Nope. Based on what I found. They are not printing anything different.